### PR TITLE
Rename custom self each

### PR DESCRIPTION
**Pull request by konstantinoskostis**

### CODE REVIEW

File: lib/resque/failure/extra.rb [hunk 1]
```diff
@@ -79,7 +79,7 @@ def _each_with_index(opts={})
         end # while
       end
 
-      def each(*args,&block)
+      def custom_each(*args,&block)
```
Comment (avgerin0s) [inline lib/resque/failure/extra.rb]: Something like `iterate_failures` seems more appropriate to me!

File: lib/resque/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Resque
-  Version = VERSION = '1.27.4.skroutz.0'
+  Version = VERSION = '1.27.4.skroutz.1.pre'
```
Comment (avgerin0s) [inline lib/resque/version.rb]: Why the `pre`?

Comment (konstantinoskostis) [reply lib/resque/version.rb]: This was an internal pre-release. Now it is `1.27.4.skroutz.1`

File: lib/resque/failure/extra.rb (hunk 1 above)
Comment (konstantinoskostis) [reply lib/resque/failure/extra.rb]: Fixed!

**Review by agis — APPROVED** [review]

**Review by dblia — APPROVED** [review]

**Pull request merged**